scripts/setup/nrfconnect/update_ncs.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import subprocess
+
+parser = argparse.ArgumentParser(description='Script helping to update nRF Connect SDK to currently recommended version.')
+parser.add_argument("-c", "--check", help="Check if your current nRF Connect SDK version is the same as recommended one.", action="store_true")
+parser.add_argument("-u", "--update", help="Update your nRF Connect SDK to currently recommended version.", action="store_true")
+args = parser.parse_args()
+
+chip_root = os.path.dirname(os.path.abspath(__file__)).replace("scripts/setup/nrfconnect", "")
```

Comment:
It's not portable. Use something like: `os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.normpath('../../..'))`